### DISCUSSION TITLE
pages

### DISCUSSION THREAD

**Discussion by nazmgm** [community/community#45541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i cant open a page with repository how can i fix it i did everthing

**Comment by radar-blip** [comment]
Open your repository, click on the `Settings` tab, click on `Pages`, and go from there. 

Edit: It seems you have some html code mixed in with your README file. 
To make more correct, use that same code and create an additional file, and name it `index.html`
